examples/dangling-else.imp

```diff
@@ -0,0 +1,9 @@
+a = 0;
+b = 0;
+
+if ( false )
+    if ( true )
+        b = 1;
+    else // is this indentation correct?
```

Comment:
Spoiler: not correct when using the PLDI version where I picked up that `avoid` on the production without `else`